src/components/SearchBar.jsx

```diff
@@ -193,6 +193,7 @@ class SearchBar extends Component {
 
   renderSuggestion = (suggestion) => (
     <div className='coz-searchbar-autosuggest-suggestion-content'>
+      {suggestion.icon && <img src={suggestion.icon} />}
```

Comment:
Can you add an `alt` attribute? Maybe something like "suggestion icon"? It's not great but better than nothing.

Reply:
Or just `alt=""`. Better nothing than a useless alternative text.

Reply:
> Better nothing than a useless alternative text.

Really though? "suggestion icon" wouldn't be entirely useless, even though it would be the same for every suggestion.
Going further — these icons aren't really content, so maybe they shouldn't be in the DOM in the first place? They might be better as background images, no?

Reply:
Nope, no background, `<img/>`is fine, better even. 
In that case, I would say that the best description to the image is the name of the file (because we don't have title, complete description or anything else about this image but the filename) and there's no need to repeat the filename in the `alt` because it's right after.
The `alt` attribute is not supposed to tell you what kind of image this is or what is its purpose but what's inside, what it represents. And we're not gonna do that. So I would say that the best value for the `alt` is no value at all.
Actually when in doubt, you should let the `alt` empty.

Reply:
Meh. Yes, `alt` should tell you what the image represents, and in this case it's "an icon for the file's mime type". So the ideal `alt` would be something like "icon of an image".
But ok, we don't have that info, so I get the leaving it blank. 

I'm still not sure `img` is better than a background image. The icons are visual enhancers, they are hardly content. But they do have a meaning so they aren't entirely visual either. Anyway I'm cool with an empty `alt`.